node/src/blockchain/blockchain_bridge.rs

```diff
@@ -1410,10 +1414,9 @@ mod tests {
                 },
             ],
         };
-        let rpc_helpers =
-            RPCHelpersMock::default().get_block_number_result(LatestBlockNumber::Err(
-                BlockchainError::QueryFailed("Failed to read the latest block number".to_string()),
-            ));
+        let rpc_helpers = LowerBCIMock::default().get_block_number_result(LatestBlockNumber::Err(
```

Comment:
I'd guess the full form of BCI, but eventually I'll be wrong. It's upto you in case you'd like to change its name. It's completely ok if you don't. 🙏

Reply:
It's been extremely tough for me to come up with a good abbreviation for BlockchainInterface

Reply:
Aha! Now, since I know the full form, everything makes sense. I think you can just add a comment where it's declared.